help/destinations/catalog/adobe/marketo-engage-person-sync.md

```diff
@@ -0,0 +1,120 @@
+
+This section describes which types of audiences you can export to this destination.
+
+| Audience origin      | Supported | Description                                                                                                                                                   |
+| -------------------- | :-------: | ------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Segmentation Service |     ✓     | Audiences generated through the Experience Platform [Segmentation Service](https://experienceleague.adobe.com/en/docs/experience-platform/segmentation/home). |
+| Custom uploads       |     ✓     | Audiences imported into Experience Platform from CSV files.                                                                                                   |
+
+## Export Type and Frequency {#export-type-and-frequency}
+
+Refer to the table below for information about the destination export type and frequency.
+
+| Item             | Type      | Notes                                                                                                                                                                                                                                                                                                                                                                                     |
+| ---------------- | --------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Export frequency | Streaming | Streaming destinations are “always on” API-based connections. As soon as a profile is updated in Experience Platform based on audience evaluation, the connector sends the update downstream to the destination platform. Read more about [streaming destinations](/help/destinations/destination-types#streaming-destinations). |
+
+{style="table-layout:auto"}
+
+## Set up Destination {#set-up-destination}
+
+>[!IMPORTANT]
+>
+>* To connect to the destination, you need the **[!UICONTROL View Destinations]** and **[!UICONTROL Manage Destinations]** [access control permissions](/help/access-control/home.md#permissions).
+
+To configure a Marketo Engage instance as a destination, that instance must be in the same IMS org as your CDP instance. If you have already configured a destination, you may select an existing Marketo account to use with your new configuration. If not, click the Connector to Destination prompt, which will allow you to set the name, description, and Marketo Munchkin ID of the desired destination. Your Marketo Instance's Munchkin ID can be found in the Admin->Munchkin menu.
```

Comment:
>  If not, click the Connector to Destination prompt, which will allow you to set the name, description, and Marketo Munchkin ID of the desired destination.

I think it's _Connect_ to destination, right? And also, where is this prompt, maybe we can say:

Navigate to the destinations catalog, find the Marketo Engage Person Sync connector, and select **Connect**